static/js/stream_popover.js

```diff
@@ -470,6 +470,13 @@ exports.register_stream_handlers = function () {
         $(".streams_popover").on("click", "a.sp-cancel", () => {
             exports.hide_stream_popover();
         });
+        if ($(window).width() <= 768) {
+            $(".popover-inner").hide().fadeIn(300);
+            $(".popover").addClass("message-info-popover");
```

Comment:
I think we want a different strategy than adding a class that's used for something else.  Probably simplest would be to do a pure CSS implementation that just lists this popover alongside `.message-info-popover` for the rules you're trying to access.

(As a general rule, we aim to not have any window width checks in JavaScript if at all possible)

Reply:
Yeah , Created a new class with similar structure specifically for <code>color-picker</code> handling responsive with media queries .